src/test/java/com/fryrank/TestConstants.java

```diff
@@ -36,7 +36,8 @@ public class TestConstants {
         TEST_TITLE_1,
         TEST_BODY_1,
         TEST_ISO_DATE_TIME_1,
-        TEST_ACCOUNT_ID
+        TEST_ACCOUNT_ID,
+        null
```

Comment:
what field does the new null represent and why is it null for each test case?

Reply:
It represents user metadata, and honestly I'm not sure why it's null, but it isn't what's being tested in the tests where this is used. I updated it to use a test one just to be explicit about what it is in [98ac0c5](https://github.com/NickPriv/FryRankBackend/pull/83/commits/98ac0c5d7f9c27e6e75a242c815fd2e47c9cbdb6).